.gitignore

```
@@ -46,7 +46,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-Pods/
+# Pods/
```

```suggestion
Pods/
```

Some of our pods haven't been updated for Swift 4.2 yet, and we've made those fixes ourselves directly in this repository while we investigate other options, so we're committing our `Pods` directory for now